force-app/main/default/classes/SOQL.cls

```diff
@@ -345,17 +410,39 @@ public with sharing class SOQL {
      */
     public static void setPrimaryContact() {
         //Use the getOpportunityDetailsForClosedWon() method to get the Opportunities
+        List<Opportunity> oppsList = SOQL.getOpportunityDetailsForClosedWon();
 
         // Create a set of Account Ids from the Opportunities
-
+        Set<Id> accountIds = new Set<Id>();
+        if(!oppsList.isEmpty()){
+            for(Opportunity opp : oppsList){
+                accountIds.add(opp.AccountId);
+            }
+        }
+        system.debug('AccountIds in the Set::: ' +accountIds);
         // Query the Contacts
-
-        // Loop through each Contact and add the Account Id and Contact to the map
-        
+        List<Contact> contactList = [SELECT Id, Name, AccountId, CreatedDate FROM Contact WHERE AccountId IN :accountIds ORDER BY Id DESC];
+        for(Contact c : contactList){
```

Comment:
Generally you'd just call this `contacts` and don't include the type of data structure like List or Set in the name.